client/components/plans/plans-compare/index.jsx

```diff
@@ -221,6 +226,12 @@ var PlansCompare = React.createClass( {
 	},
 
 	render: function() {
+		var compareString = this.translate( 'Compare Plans' );
+
+		if ( this.props.selectedSite.jetpack ) {
```

Comment:
We should check if `this.props.selectedSite` is defined: `this.props.selectedSite && this.props.selectedSite.jetpack`